test/unittests/libweb3jsonrpc/jsonrpc.cpp

```diff
@@ -319,6 +336,90 @@ BOOST_AUTO_TEST_CASE(eth_sendTransaction)
     BOOST_CHECK_EQUAL(txAmount, balance2);
 }
 
+BOOST_AUTO_TEST_CASE(eth_sendRawTransaction)
+{
+    auto senderAddress = coinbase.address();
+    auto receiver = KeyPair::create();
+    
+
+    BOOST_TEST_CHECKPOINT("Sending a signed transaction from 0 balance account should fail");

```

Comment:
I'd prefer them to be separate tests instead of several test cases separated by checkpoints, to make them more isolated, but that's not critical.